new-docs/content/any/getting-started/policies/index.md

```diff
@@ -9,325 +9,438 @@ description: Learn about writing Oso policies - the source of truth for authoriz
+as a simple declarative sentence of English (or other natural language),
+it should be straightforward to turn into a Polar rule.
+
+If you're unsure of what _kind_ of authorization policy to write,
+you may want to head over to the [Conceptual Guides](learn) or
+start with a familiar pattern such as [roles](/guides/roles).
+This tutorial will not be concerned with a specific kind of policy,
+but rather with the language in which many different kinds of
+policies may be expressed.
+
+## Application Setup
+
+To use Oso for authorization, your application must first:
+
+1. Load the Oso library: `from oso import Oso`
+2. Create an `Oso` instance: `oso = Oso()`
+3. Load a policy: `oso.load_file(policy)`.
+
+Then, at authorization time, you can call:
 
-The syntax might feel a bit foreign at first, but fear not: almost anything you
-can express in imperative code can equally be expressed in Polar — often more
-concisely and closer to how you might explain the logic in natural language.
-
-{{% callout "Note" "green" %}}
-Policies are stored in Polar files (extension `.polar`), which are loaded
-into the authorization engine using the [Oso library](reference).
-{{% /callout %}}
-
-## Rule basics
+```python
+oso.is_allowed(actor, action, resource)
+```
 
-Policies are made up of [rules](polar-syntax#rules). Each rule defines a
-statement that is either true or false. Oso answers queries by evaluating rules
-that match the query name and parameters. Let’s take a basic [allow
-rule](glossary#allow-rules) as an example:
+This returns a boolean indicating whether the given `actor`
+is authorized to perform `action` on `resource` according to
+the current policy.
 
-```polar
-allow(actor, action, resource);
-```
+The arguments to `oso.is_allowed` may be arbitrary {{< lang >}}
+objects. We'll often use strings for examples, but in your
+application they could be ORM model objects, URIs, numbers, etc.
 
-<!-- TODO(gj): link `Oso.is_allowed()` once API docs are hooked up. -->
-When we use `Oso.is_allowed()` (or equivalent), we are making a query that asks
-Oso to evaluate all rules that match *(a)* on the rule name (`allow`), and
-*(b)* on all the inputs.
+## Rules
 
-In the rule above, `actor`, `action`, and `resource` are simply the parameter
-names, i.e., they are variables that will match *anything*.
+The way you control _which_ boolean value is returned by
+`oso.is_allowed` is to define and load Polar rules that
+_match_ only the desired set of `actor`, `action`, and `resource`
+arguments. This matching may involve logical connectives,
+type checks, equality checks, variable bindings, field lookups,
+method calls, arithmetic, comparisons, etc. See the
+[Polar Syntax guide](polar-syntax) for a complete list
+of available operators.
 
-But if we replace `action` with a concrete type…
+Let's start with the basic syntax. A _rule definition_ in Polar
+has a _name_, a _parameter list_, and an optional _body_. Here's
```

Comment:
In general I think it's more clear to use **bold face** for keyword-type emphasis (as here), and _italics_ for emphasis. When I see lots of italics, I read it as if someone is speaking very deliberately and emphasizing each word.

Reply:
Ok. I have a very academic habit of emphasizing the first occurrence of every new keyword or technical term, but don't have strong feelings on what kind of emphasis to use. Switched to bold so we can use italics for ordinary emphasis.